### PR TITLE
[MH2018-17] --> [MH-Team-2-Shopify] bugFix/breadcumbs

### PR DESCRIPTION
**Pull request by luisarmandodv**
### Now, it has linkable capabilities.
captura de pantalla 2018-04-02 a la s 17 31 39

## For example:
captura de pantalla 2018-04-02 a la s 17 32 38

### CODE REVIEW

File: Lil-Peep/src/styles/modules/site-collection.scss [hunk 1]
```diff
@@ -77,6 +77,7 @@
 
     .home-route-section {
       color: #{$home-route-section};
```
Comment (rubiocanino) [inline Lil-Peep/src/styles/modules/site-collection.scss]: That can be without the #{} -> $home-route-section

Comment (Ilyeo) [inline Lil-Peep/src/styles/modules/site-collection.scss]: can you remove `#{}` from variables, please?

Comment (luisarmandodv) [reply Lil-Peep/src/styles/modules/site-collection.scss]: Got it

Comment (albertm805) [inline Lil-Peep/src/styles/modules/site-collection.scss]: Remove the `#{` as @rubiocanino said. Just leave the `$home-route-section`

Comment (luisarmandodv) [reply Lil-Peep/src/styles/modules/site-collection.scss]: Got it.

File: Lil-Peep/src/styles/modules/site-collection.scss [hunk 2]
```diff
@@ -72,22 +72,23 @@
...
+      color: $categories-route-section;
     }
   }
 }
 
 .categories-view {
-  background-color: #{$background-color};
-  font-family: 'Oswald', sans-serif;
+  background-color: $background-color}
+  font-family: $oswald-font;
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/site-collection.scss]: Syntax Error: Invalid CSS after "...y: $oswald-font": expected "{", was ";"

File: Lil-Peep/src/styles/modules/site-collection.scss [hunk 3]
```diff
@@ -231,6 +232,18 @@
       background-color: $on-hover-button;
       cursor: pointer;
     }
+
+    a {
+      text-decoration: none;
+    }
+
+    h4 {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/site-collection.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: Lil-Peep/src/styles/modules/site-collection.scss [hunk 4]
```diff
@@ -231,6 +232,18 @@
       background-color: $on-hover-button;
       cursor: pointer;
     }
+
+    a {
```
Comment (magmahound) [inline Lil-Peep/src/styles/modules/site-collection.scss]: Selector should have depth of applicability no greater than 2, but was 3

**Pull request merged**